src/steps/10-FinancialDetails/IGCE/SurgeCapacity.vue

```diff
@@ -1,15 +1,100 @@
 <template>
-  <div>
-    Future page for Surge Capacity
-  </div>
+  <v-container fluid class="container-max-width mb-7">
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header mb-3">
+          Do you want to include surge capacity within the scope of your
+          contract?
+        </h1>
+        <div class="copy-max-width">
+          <p id="IntroP" class="mb-10">
```

Comment:
pls change `mb-10` to `mb-7` - the radio group has a top margin which makes too much white space with mb-10